android/WALT/app/src/main/java/org/chromium/latency/walt/FrontPageFragment.java

```diff
@@ -42,7 +41,7 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
         View view = inflater.inflate(R.layout.fragment_front_page, container, false);
 
         if (container.getContext().getPackageManager().
-                hasSystemFeature(PackageManager.FEATURE_MIDI)) {
+                hasSystemFeature("android.software.midi")) {
             view.findViewById(R.id.action_midi).setVisibility(View.VISIBLE);
```

Comment:
Probably as a separate commit later.
Let's change this so it's always visible but grayed out or somehow disabled if irrelevant. 
As an option we can just show a toast message saying midi is not available when this is clicked.
Otherwise it's really weird to see the UI subtly different without knowing why.